diff --git a/diagrams/db_schema.puml b/diagrams/db_schema.puml
index 60b4df4..46b3838 100644
--- a/diagrams/db_schema.puml
+++ b/diagrams/db_schema.puml
@@ -19,8 +19,8 @@ entity config {
   round bigint
 }
 
-entity Order Book {
-  *PK_Order Book_id : int <<generated>>
+entity orderbook {
+  *PK_orderbook_id : int <<generated>>
   --
   *round : bigint unsigned
   *appid : bigint unsigned
@@ -51,5 +51,5 @@ entity trade_history {
 }
 
 trade_history }|..|| asset_info
-Order Book }|..|| asset_info
+orderbook }|..|| asset_info
 @enduml
